blueprints/unifi/install.sh

```diff
@@ -34,7 +34,7 @@ UP_USER="${!UP_USER:-$1}"
 UP_PASS="jail_${1}_up_password"
 INCLUDES_PATH="${SCRIPT_DIR}/blueprints/unifi/includes"
 
-if [ -z "${!DB_PASSWORD}" ]; then
+if [ -z "${DB_PASS}" ]; then
```

Comment:
Shoot my mistake.
Still needs to be:
```suggestion
if [ -z "${!DB_PASS}" ]; then
```

Reply:
yea, my bad. fixed